heartbeat/sg_persist.in

```diff
@@ -495,14 +495,26 @@ sg_persist_action_monitor() {
 
     if [ ${#REGISTERED_DEVS[*]} -eq 0 ]; then
         ocf_log debug "$RESOURCE monitor: no registrations"
+        if [ -n "$ACT_MASTER_SCORE" ]; then
+	    ocf_run $MASTER_SCORE_ATTRIBUTE --delete
+	    ocf_run $PENDING_ATTRIBUTE --delete
+	fi
```

Comment:
You should indent your fi's to the same level as the if's for better readability, and use 4 spaces per indent as the rest of the code in this agent.

Reply:
Indeed. Very strange. I just checked my source. All aligned.
```
    if [ ${#REGISTERED_DEVS[*]} -eq 0 ]; then
        ocf_log debug "$RESOURCE monitor: no registrations"
        if [ -n "$ACT_MASTER_SCORE" ]; then
            ocf_run $MASTER_SCORE_ATTRIBUTE --delete
            ocf_run $PENDING_ATTRIBUTE --delete
        fi
        return $OCF_NOT_RUNNING
    fi

```

Reply:
Okey, I see. Looked like I used tab sometimes. I'll change it for space.